lib/src/drawings.dart

```diff
@@ -151,6 +216,8 @@ class TextDrawing extends GYWDrawing {
       font: font,
       size: data["size"] as int?,
       color: data["color"] as String?,
+      maxWidth: data["maxWidth"] as int?,
+      maxLines: data["maxLines"] as int?,
```

Comment:
```suggestion
      maxWidth: data["max_width"] as int?,
      maxLines: data["max_lines"] as int?,
```

I think I use `lower_case` keys in JSON in other objects. Can you check ?